docs/lib/datastore/fragments/native_common/getting-started.md

```diff
@@ -102,7 +97,7 @@ To write to the database, create an instance of the `Post` model and save it.
 <inline-fragment platform="js" src="~/lib/datastore/fragments/js/getting-started/60_saveSnippet.md"></inline-fragment>
 <inline-fragment platform="ios" src="~/lib/datastore/fragments/ios/getting-started/60_saveSnippet.md"></inline-fragment>
 <inline-fragment platform="android" src="~/lib/datastore/fragments/android/getting-started/60_saveSnippet.md"></inline-fragment>
-<inline-fragment platform="flutter" src="~/lib/datastore/fragments/flutter/getting-started/60_saveSnippet.md"></inline-fragment>
+<inline-fragment platform="flutter" src="~/lib/datastore/fragments/flutter/getting-started/80_saveSnippet.md"></inline-fragment>
```

Comment:
Nit: What happened to 70? 😅 

Imo, it's a little confusing (and perhaps unnecessary?) to pull these numbers out of sync with their platform equivalents.